website/source/docs/agent/cloud-auto-join.html.md

```diff
@@ -373,3 +373,6 @@ $ consul agent -retry-join "provider=k8s label_selector=\"app=consul,component=s
   set, it defaults to all namespaces.
 - `label_selector` (optional) - the label selector for matching pods.
 - `field_selector` (optional) - the field selector for matching pods.
+
+Kubernetes token used by the provider needs to have permissions to list pods
```

Comment:
```suggestion
Note: The Kubernetes token used by the provider needs to have permissions to list pods
```

Reply:
We're not saying `Note` for other providers, so I kept it as is for consistency but added `The` as you suggested. Thanks!